extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/OtTracerPropagator.java

```diff
@@ -66,6 +70,21 @@ public static OtTracerPropagator getInstance() {
         spanContext.getTraceIdAsHexString().substring(TraceId.getHexLength() / 2));
     setter.set(carrier, SPAN_ID_HEADER, spanContext.getSpanIdAsHexString());
     setter.set(carrier, SAMPLED_HEADER, String.valueOf(spanContext.isSampled()));
+
+    Baggage baggage = Baggage.fromContext(context);
+    if (!baggage.isEmpty()) {
+      baggage.forEach(
+          (key, value, metadata) -> {
+            String metadataValue = metadata.getValue();
+            String baggageValue;
+            if (metadataValue != null && !metadataValue.isEmpty()) {
+              baggageValue = value + ";" + metadataValue;
```

Comment:
Metadata is not supported for OT, similarly to Jaeger's format ;(

Reply:
I removed metadata from inject.
Should I also remove it from extract?

Reply:
Yes, please ;)

Reply:
done